internal/pkg/utils/etcdlogger/etcdlogger.go

```diff
@@ -213,12 +213,14 @@ func quote(v string) string {
 }
 
 func keyToStr(key, end []byte) string {
-	if len(key) == 0 {
+	keyStr := strings.ReplaceAll(string(key), "\000", "<NUL>")
+	endStr := strings.ReplaceAll(string(end), "\000", "<NUL>")
```

Comment:
Etcd client interne pouziva pre iteraciu vsetkych klucv range `[]byte{0}, []byte{0}`, `\0` (alebo inak aj `NUL` char) sa nevypisuje.
Aby sa to dalo v testoch asertovat, tak tuto specialnu hodnotu nahradime.